CHANGELOG.md

```diff
@@ -1,8 +1,29 @@
-# dbt_youtube_analytics v0.UPDATE.UPDATE
+# dbt_youtube_analytics v0.4.0
+
+## 🚨 Breaking Changes 🚨 (within the upstream [dbt_youtube_analytics_source v0.4.0 release](https://github.com/fivetran/dbt_youtube_analytics_source/releases/tag/v0.4.0)):
+- Removed support for the `video_metadata` Cloud Function generated source table and downstream models. This also means the following variables are no longer used or necessary within the package: ([PR #12](https://github.com/fivetran/dbt_youtube_analytics_source/pull/12))
+  - `youtube__using_video_metadata`
+  - `youtube_metadata_schema`
+  - `youtube_analytics_database`
+- To be consistent with our other packages, the identifier variables have been updated. Please see the following changes to the identifier variables used in this package. ([PR #12](https://github.com/fivetran/dbt_youtube_analytics_source/pull/12))
+
+| **old identifier name** | **new identifier name** |
+| ------------------------|-------------------------|
+| `youtube__channel_basic_table` | `youtube_analytics_channel_basic_a_2_identifier` |
+| `youtube__channel_demographics_table` | `youtube_analytics_channel_demographics_a_1_identifier` |
+
+## Feature Updates:
+- Following the latest update to **all** Fivetran YouTube Analytics connectors, the `video` metadata is now synced as part of the connector by default. Therefore, the `video` source and staging models along with the following downstream models have been updated to reference the `video` source table which is synced by default: ([PR #13](https://github.com/fivetran/dbt_youtube_analytics/pull/13))
+  - `youtube__video_metadata`
+  - `youtube__video_report`
+  - `youtube__age_demographics_pivot`
+  - `youtube__demographics_report`
```

Comment:
We changed the name of one field in `youtube__demographics_pivot`, should we call that out in the CHANGELOG or is that superfluous?

Reply:
What field did we rename?

Reply:
The field names seem to be the same. I agree if we renamed a field we should call it out, but unsure what field you are referring to.

![image](https://github.com/fivetran/dbt_youtube_analytics/assets/74217849/4e2c1db1-7363-4d4d-9084-61a380a62ece)

Reply:
This is a screenshot of `youtube__gender_demographics_pivot`. In `youtube_demographics_pivot` you renamed `publisehd_at` to `video_published_at` ([you updated it in the yml as well](https://github.com/fivetran/dbt_youtube_analytics/pull/13/files#diff-8fc3725d060a4b921fc6ffdcfa810a09af24ee2b6d11cf8967172a81adc66d2cR24))

Reply:
Actually looks like you did that in `youtube__video_report` as well.  Looks like it was `published_at` from `stg_youtube__video_metadata` and it's now `video_published_at` from `stg_youtube__video`, based off of `snippet__published_at` from the `video` source.

You could probably just mention you renamed the `published_at` field to `video_published_at` in the source package and it's pushed through the downstream models.